pages/status.tsx

```diff
@@ -6,12 +6,18 @@ import ErrorSummary, {
   getErrorSummaryItem,
 } from '../components/ErrorSummary'
 import Layout from '../components/Layout'
-import { CheckStatusReponse, CheckStatusRequestBody } from './api/check-status'
+import { CheckStatusRequestBody } from './api/check-status'
 import { useCheckStatus } from '../hooks/api/useCheckStatus'
 import { useFormik } from 'formik'
 import * as Yup from 'yup'
 import useTranslation from 'next-translate/useTranslation'
 import StatusInfo from '../components/StatusInfo'
+import { GetStaticProps } from 'next'
+import Router from 'next/router'
+
+export const getStaticProps: GetStaticProps = async () => ({
```

Comment:
I suggest to move `getStaticProps` before `export default Status` as you would see in next.js examples.

Reply:
https://nextjs.org/docs/basic-features/data-fetching/get-static-props#using-getstaticprops-to-fetch-data-from-a-cms

Reply:
I honestly hate that that's how examples look; its so very backwards as this function is always called before the actual component. It's far more logical to keep it at the top.

Reply:
I see your point but it's a convention that you will see everywhere with next.js. I like to follow next.js convention.

Reply:
okay.... :(